solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -31,7 +31,8 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
 
     uint256 private __reserveBalances;
     uint256 private __reserveBalancesProd;
-    IERC20[] private __reserveTokens;
+    IERC20[2] private __reserveTokens;
+    uint16 __reserveTokensLength;
```

Comment:
Can you try to use an `immutable` or hardcoded 2 instead of a storage variable?

Reply:
I don't understand.
Have you followed variable `__reserveTokensLength` in the code? It can be anything between 0 and 2.

Reply:
Ah, I remembered that we are always creating it with two reserves from the get-go.

Reply:
@barakman any reason why it's not `_reserveTokensLength` according to our convention?

Reply:
I don't understand the question.

Reply:
__reserveTokensLength should be _reserveTokensLength

Reply:
done